zadanie_15

annbla7 opened the pull request.

annbla7 committed:
zadanie_15

kadyb commented on zadania/zadanie_15.py:
```diff
@@ -0,0 +1,25 @@
+
+#Odległość sferyczna (1315.51 km) jest najniższa, ponieważ zakłada, że Ziemia jest idealną kulą.
+#Odległość elipsoidalna (1316.2 km) jest najdokładniejsza, biorąc pod uwagę rzeczywisty kształt Ziemi.
+#Odległość Haversine (1449.78 km) jest największa, ponieważ metoda ta, mimo że jest bardziej precyzyjna od zwykłej sferycznej, wciąż nie uwzględnia elipsoidalnego kształtu Ziemi.
```
W którymś miejscu pomylona jest długość z szerokością geograficzną, i stąd niepoprawny wynik.

annbla7 replied:
Dziękuję za zwrócenie uwagi. Po sprawdzeniu jeszcze raz wzoru znalazłam błąd. Zostało to poprawione.

annbla7 committed:
Poprawione zadanie_15.py

Poprawienie błędu - zamienienie długości z szerokością geograficzną, aby pokazywał się poprawny wynik.
